### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/a90c582048ccef49cbbecdd1a6a79be624bcc83b

### CODE REVIEW

File: en/firefox/new/desktop.ftl [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-get-firefox-android = Get { -brand-name-firefox-browser } for <strong>{ -brand-name-android }</strong>
+firefox-desktop-download-get-firefox-ios = Get { -brand-name-firefox-browser } for <strong>{ -brand-name-ios }</strong>
+firefox-desktop-download-download-the-mobile = Download the { -brand-name-firefox } mobile browser for automatic protection on all your devices.
+firefox-desktop-download-learn-about-the = Learn about the { -brand-name-firefox } desktop browser below
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-from-security-to = From security to news to gaming, there’s an <a { $attrs }>extension for everyone</a>. Add as many as you want until your browser is just right.
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:119]: Please add a comment on what it means `your browser is just right`. What is the alternative?

File: en/firefox/new/desktop.ftl [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/about/
+firefox-desktop-download-firefox-was-created = { -brand-name-firefox } was created by <a {$attrs }>{ -brand-name-mozilla }</a> as a faster, more private alternative to browsers like Internet Explorer, and now { -brand-name-chrome }. Today, our mission-driven company and volunteer community continue to put your privacy above all else.
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:123]: Brand: `Internet Explorer`

File: en/firefox/new/desktop.ftl [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-questions = Questions? <a { $attrs }>{ -brand-name-mozilla } support</a> has you covered.
+
+# The phrase “Now get even more from Firefox” is in specific reference to signing up for an account, which unlocks access to all our new products and services.
+firefox-desktop-download-youve-already-got-the-browser = You’ve already got the browser. Now get even more from { -brand-name-firefox }.
+firefox-desktop-download-watch-for-hackers-with = Watch for hackers with { -brand-name-firefox-monitor }, protect passwords with { -brand-name-firefox-lockwise }, and more.
+firefox-desktop-download-get-more-from-firefox = Get More From { -brand-name-firefox }
+firefox-desktop-download-just-download-the-browser = Just Download The Browser
+
+## URL: https://www-dev.allizom.org/firefox/new/thanks/
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:151]: I can't see the page. Any alternative link for review?

File: en/firefox/new/desktop.ftl [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:153]: Nothing is being highlighted in the string `Almost there!`

File: en/firefox/new/desktop.ftl (hunk 3 above)
Comment (craigcook) [reply en/firefox/new/desktop.ftl:151]: This should be /firefox/download/thanks/

**Comment by craigcook** [comment]
Stephanie made the fixes and the robot opened a fresh PR: https://github.com/mozilla-l10n/www-l10n/pull/84

Closing this one in favor of that one.

**Pull request closed without merging**